datasetinsights/datasets/synthetic.py

```diff
@@ -346,7 +302,7 @@ def download_captures_from_manifest(self, manifest_file):
         Args:
             manifest_file (str): path to a manifest file
         """
-        _download_captures(self.root, manifest_file)
+        _download_captures(self.data_path, manifest_file)
```

Comment:
This code path is actually dead and should be removed. `self.download` is no longer called in `__init__`.

Reply:
I don't think adding things to refactor to this PR is beneficial. In this PR the code path is not dead.